_posts/2017-07-25-oes-case-study.md

```diff
@@ -0,0 +1,41 @@
+---
+title: A search for efficiency led the Evaluation Sciences team to the U.S. Web Design Standards
+tags:
+- web design standards
+- design
+- case study
+- design
+---
+The [U.S. Web Design Standards](https://standards.gsa.gov) are currently being used on hundreds of government sites, with an audience of more than 59 million monthly users. In this seventh  post in our series, we sat down with Maureen Earley, program management specialist with the Office of Evaluation Sciences (OES), an office within the General Services Administration, to talk about her team’s use of the Standards on [oes.gsa.gov](https://oes.gsa.gov). The OES brings diverse scientific experts onto their team for year-long rotations, where they work with federal agencies to implement and rigorously test changes made to government programs to evaluate whether or not they increase the ability of agencies to achieve their missions with greater success and efficiency.
+
+**Standards team:** How did you hear about the U.S. Web Design Standards?
+
+**Maureen Earley:**  Members of our team  worked with the 18F Federal Front Door team that was involved in creating the Standards. The Office of Evaluation Sciences is a champion of the Standards because we’re committed to making government easier for people to interact with. We also wanted to make sure our own website was an ambassador of the Standards and all that they stand for.
+
+**Standards team:** Why did you decide to use the Standards?
+
+**Maureen Earley:** We understood why the Standards were created and why good design across government is important. We also didn’t want to recreate the wheel in designing our website. For example, we realized during the design process that our OES logo didn’t meet color contrast requirements for accessibility. But, because we were referencing the Standards we were able to avoid this issue. We wanted to defer to the Standards and the expertise that already exists.
+
+We also have been working with the team running [Federalist](https://federalist.fr.cloud.gov), the open source static website publishing service, building out templates that other teams across government can use. For example, our [Team page](https://oes.gsa.gov/team/) is now a Federalist template that anyone can use.
+
+When we talk with other offices within government who want to redesign their sites, they get very excited about the Standards, and the Federalist platform, because someone has already done the work for them. It makes their job and life a lot easier.
+
+**Standards team:** How did you integrate your work with the Standards?
+
+**Maureen Earley:** We worked with one of 18F’s developers who was doing the various coding elements for us. She redid the back-end of our site to be compliant with the Standards.
```

Comment:
I looked up the [project](https://github.com/gsa-oes/office-of-evaluation-sciences) and see that it is a Jekyll site so I'm going to assume they meant the template logic and going to rework that sentence and maybe throw the jekyll name in to clarify.